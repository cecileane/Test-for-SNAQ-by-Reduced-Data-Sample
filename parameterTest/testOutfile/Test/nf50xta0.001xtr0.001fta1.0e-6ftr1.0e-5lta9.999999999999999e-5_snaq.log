optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=1.0e-6,
                       xtolAbs=0.001, xtolRel=0.001.
 max number of failed proposals = 50, liktolAbs = 9.999999999999999e-5.
rootname for files: nf50xta0.001xtr0.001fta1.0e-6ftr1.0e-5lta9.999999999999999e-5_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 45838
seed: 45838 for run 1
Sat Oct  1 17:29:01 2016

 BEGIN SNaQ for run 1, seed 45838 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 89 (failures: 50) with -loglik=126.74251 and ht_min=[0.2856,10.0,0.48942,0.51916,1.51213]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 26		 26		 0	 |	 0		 0		 --
mvtarget	 20		 20		 2	 |	 0		 0		 --
chdir		 13		 13		 3	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 29		 22		 2	 |	 0		 0		 --
Total		 89		 82		 8	 |	 0		 0		 0
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 0.0		 0.0

 FINISHED SNaQ for run 1, -loglik of best 126.74250981976661
((4,(2,(1)#H7:::0.7143969724129189):1.5121332598646375):0.4894187563530739,(5,(6,#H7:::0.2856030275870811):10.0):0.5191611334338524,3);
---------------------
seed: 51002 for run 2
Sat Oct  1 17:29:07 2016

 BEGIN SNaQ for run 2, seed 51002 and hmax 1
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 84 (failures: 50) with -loglik=126.74251 and ht_min=[0.2856,10.0,0.48942,0.51916,1.51213]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 18		 18		 1	 |	 0		 0		 --
mvtarget	 35		 35		 2	 |	 0		 0		 --
chdir		 10		 10		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 20		 12		 1	 |	 8		 8		 --
Total		 84		 76		 6	 |	 8		 8		 3
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 0.4

 FINISHED SNaQ for run 2, -loglik of best 126.74250981974933
(4,(2,(1)#H7:::0.7143969525162241):1.5121336142196067,(3,(5,(6,#H7:::0.2856030474837759):10.0):0.5191607563925765):0.4894188345854887);
---------------------

Sat Oct  1 17:29:13 2016
MaxNet is (4,(2,(1)#H7:::0.7143969525162241):1.5121336142196067,(3,(5,(6,#H7:::0.2856030474837759):10.0):0.5191607563925765):0.4894188345854887); 
with -loglik 126.74250981974933
