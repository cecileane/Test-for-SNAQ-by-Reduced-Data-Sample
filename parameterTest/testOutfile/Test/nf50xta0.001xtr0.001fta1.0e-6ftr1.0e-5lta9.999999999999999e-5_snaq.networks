(4,(2,(1)#H7:::0.7143969525162241):1.5121336142196067,(3,(5,(6,#H7:::0.2856030474837759):10.0):0.5191607563925765):0.4894188345854887);, with -loglik 126.74250981974933 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(4,(2,(#H7:::0.31182652654477044,1):2.660116121610698):0.6594702812442145,(3,(5,(6)#H7:::0.6881734734552296):1.5288733215768175):0.49796136349939524);, with -loglik 128.82426697960568
(3,(5,(6,(1,(#H7:::0.40092695007596746,2):1.8874023999285023):1.167465158908969):0.2215269494437222):0.021351355953119374,(4)#H7:::0.5990730499240325);, with -loglik 173.5927186009034
(4,#H7:::0.43550149823227724,(3,(5,(6,(1,(2)#H7:::0.5644985017677228):5.580898503426651):0.15751443403232562):0.028513552414984693):1.7246267770268637);, with -loglik 191.94992973295894
(4,(2,((6,(#H7:::0.404314287601392,5):9.977301160967386):0.9917155037100389,1):0.09556007875516255):0.07118423001916974,(3)#H7:::0.595685712398608);, with -loglik 215.12901337353458
(4,(2,((6,#H7:::0.49092481622849204):5.4467547220274914,1):0.045520890468163905):0.20535525250909345,(3,(5)#H7:::0.509075183771508):1.149806256222478);, with -loglik 231.18102764185895
