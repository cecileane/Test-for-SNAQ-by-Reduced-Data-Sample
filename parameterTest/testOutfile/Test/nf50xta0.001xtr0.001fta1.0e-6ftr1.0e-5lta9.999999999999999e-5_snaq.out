(4,(2,(1)#H7:::0.7143969525162241):1.5121336142196067,(3,(5,(6,#H7:::0.2856030474837759):10.0):0.5191607563925765):0.4894188345854887); -Ploglik = 126.74250981974933
 Dendroscope: (4,(2,(1)#H7):1.5121336142196067,(3,(5,(6,#H7):10.0):0.5191607563925765):0.4894188345854887);
 Elapsed time: 12.781222557 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (4,(2,(1)#H7:::0.7143969525162241):1.5121336142196067,(3,(5,(6,#H7:::0.2856030474837759):10.0):0.5191607563925765):0.4894188345854887);, with -loglik 126.74250981974933
 ((4,(2,(1)#H7:::0.7143969724129189):1.5121332598646375):0.4894187563530739,(5,(6,#H7:::0.2856030275870811):10.0):0.5191611334338524,3);, with -loglik 126.74250981976661
-------
